spec/features/agent_manages_appointments_spec.rb

```diff
@@ -10,6 +10,18 @@
       when_they_want_to_book_an_appointment
       and_they_fill_in_their_appointment_details
       then_that_appointment_is_created
+      and_the_customer_gets_an_email_confirmation
+    end
+  end
+
+  context 'Customer does not give an email' do
```

Comment:
Can we flatten these contexts? When they only have a single context (and in a feature-style spec) there isn't any point separating them.